### ISSUE TITLE
Adding a readiness probe to the AMQP - Kafka bridge

### ISSUE THREAD

**Issue by ppatierno** [EnMasseProject/enmasse#83]
When the images aren't available locally, it could happen that the AMQP - Kafka bridge image is download before the messaging one. In this case with the current liveness probe the bridge pod is restarted a lot of times because the messaging isn't up and running yet and it can't connect as client.
It's related to another problem in the bridge which start the HTTP server (for the liveness probe) after such connection is established (see https://github.com/EnMasseProject/amqp-kafka-bridge/issues/73).
I'm going to move the HTTP server startup before the connection with messaging is established so that the pod can reply to the liveness probe but then provide another HTTP path (/ready ?) when the connection is established. Such path can be tested by the readiness probe.
In summary : 
- the liveness probe is used just to check if the bridge is live even if it's not still connected to the messaging system
- the readiness probe works when the bridge is connected and is effectivily ready to receive

**Comment by ppatierno** [comment]
Close on behalf of 32b05209e41292cf6606e2b89dfa7ff9467795e6
